gluten-core/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala

```diff
@@ -454,9 +454,9 @@ object ExpressionConverter extends SQLConfHelper with Logging {
           m
         )
       case rand: Rand =>
-        BackendsApiManager.getSparkPlanExecApiInstance.genRandTransformer(
+        GenericExpressionTransformer(
           substraitExprName,
-          replaceWithExpressionTransformerInternal(rand.child, attributeSeq, expressionsMap),
+          Seq(replaceWithExpressionTransformerInternal(rand.child, attributeSeq, expressionsMap)),
```

Comment:
I wonder if we still need this specialization as the default branch could produce a GenericExpressionTransformer.

Reply:
good suggestion!